action/__main__.py

```diff
@@ -33,6 +33,7 @@
             database=os.environ["DB_NAME"],
             user=os.environ["DB_USER"],
             password=os.environ["DB_PASSWORD"],
+            exclude_tables=os.getenv("EXCLUDE_TABLES","").split(","),
```

Comment:
Splitting an empty string produces a list of one empty string (i.e. `['']`), whereas your intention is to produce an empty list